.github/ISSUE_TEMPLATE/GITHUB_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+---
+name: '❔ GitHub Request'
+about: 'Request update to permission / management / settings on opensearch-project GitHub organization and its repositories'
+title: '[GitHub Request]'
+labels: 'github-request'
+assignees: ''
+---
+
+### What is the type of request?  
+* _User Permissions (e.g., organization membership, repository access)_  
+* _Repository Management (e.g., creation, deletion, settings updates)_  
+* _GitHub Actions (e.g., higher-spec runners, GPU-enabled runners, troubleshooting)_  
+* _Others_
+
+### Where does this request apply?  
+* _Repository_  
+* _Organization_
+
+### Details of your request  
+_Describe the actions you are requesting (e.g., GitHub Actions in the`<repository> take too long to allocate runners)._
+
+### If this is about a repository, did you contact the maintainer first?  
+_If yes, explain why you are opening this request after consulting with the maintainer._
+
+### Additional information to support your request  
+_Provide reasons for your request. For example, if you request to add a new maintainer, confirm you followed the steps in the [Becoming a Maintainer](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#becoming-a-maintainer) guide._
```

Comment:
Do you think its worth making a separate template for that maintainer request type as it seems common

Reply:
Was thinking the same. Tho at the moment seems like this template could cover all of these cases in one take so I am a bit hesitate.